k8s.tf

```diff
@@ -61,6 +66,17 @@ data "aws_eks_node_group" "gpuivsnodes" {
   node_group_name = replace(module.eks.managed_node_groups[0]["gpuivsnodes"]["managed_nodegroup_id"][0], "${local.infrastructurename}:", "")
 }
 
+resource "aws_autoscaling_group_tag" "default_node-template_resources_ephemeral-storage" {
+  autoscaling_group_name = data.aws_eks_node_group.default.resources[0].autoscaling_groups[0].name
+
+  tag {
+    key   = "k8s.io/cluster-autoscaler/node-template/resources/ephemeral-storage"
+    value = "${var.linuxNodeDiskSize}G"

```

Comment:
nicht "Gi" ? In der Variablenbeschreibung steht zumindest "... in GiB"

Reply:
Laut https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/aws/README.md#auto-discovery-setup ist "G" richtig.